### ISSUE TITLE
Pre-training model selection for testing

### ISSUE THREAD

**Issue by daydreamer2024** [jaewon-lee-b/lte#17]
Hello, I would like to ask why the epoch_last model is selected as the model to be imported for the final test in the arbitrary scale super-resolution task, but not the epoch_best?

**Comment by jaewon-lee-b** [comment]
Hello @daydreamer2024 

The epoch-best model does not guarantee the best performance for all benchmark datasets. In fact, the epoch-best is determined by a few samples from the validation set.

In practice, after around 850 iterations, the model sufficiently  converges. Therefore, picking any model among converged is not a problem, including last one.

But I just picked last one because I did another thing during trainig. 😀

**Comment by daydreamer2024** [comment]
Thanks for your answer. I noticed by looking at the logs during training that on the last 200 epochs. The loss only fluctuates within 0.001 and the val psnr fluctuates a lot, even up to 0.5. The val psnr is not showing a tendency to increase, what is causing this problem. 

Also can I understand that the smaller the loss during training may produce better test results, the val psnr results have little to do with the test results.

**Comment by jaewon-lee-b** [comment]
Val PSNR fluctuation is not a problem but is relatively standard. PSNR is calculated by comparing randomly sampled data in a validation set. It is sufficient to check whether Val PSNR increases compared to 1st iteration.

Low loss guarantees good PSNR performance when other conditions, such as the downsampling model or dataset, are maintained if there is no overfitting issue. But low Val PSNR does not guarantee anything🥲
